src/coreComponents/constitutive/solid/Damage.hpp

```diff
@@ -86,6 +86,7 @@ class DamageUpdates : public UPDATE_BASE
   using UPDATE_BASE::hypoUpdate_StressOnly;
   using UPDATE_BASE::hyperUpdate;
   using UPDATE_BASE::hyperUpdate_StressOnly;
+  using UPDATE_BASE::saveConvergedState;
```

Comment:
These changes to saveConvergedState() allow us to call these routines within a single RAJA loop, rather than having to do separate launches for strain updates and saving states.   Currently, there are then two saveConvergedState() methods available: one which is array-wise in SolidBase, and one which is point-wise (called with a specific element / quadrature point index) within SolidBaseUpdates.  I need to find a way to make sure one calls the other, but there is a minor hiccup in doing so.  For the moment, there is a risk the two methods could diverge from one another if the developer is not careful.  I'll think a bit how to fix this.  If we can, it will allow several kernel launches to be merged into one.